### PR TITLE
Help menu

### PR DESCRIPTION
**Pull request by Tensuko**
New topics for the help menu.

### CODE REVIEW

File: translations/translation_en.xml [hunk 1]
```diff
@@ -480,24 +480,24 @@ Working on vines is a bit more complex then working on a normal field.
...
-A Vine course should be generated on the vines, as it got to drive and work on the vines.
+A vine course should be generated on the vines, as it got to drive and work on the vines.
 		"/>
 		<text name="CP_help_page_vineWorkPruner_text"													text="
 The pre-pruner works on the vines, so the course have to be generated on the vines.
 The Tools comes with an offset for the tractor, so the tractor drives between the vines.
 		"/>
 		<text name="CP_help_page_vineWorkSprayer_text"													text="
-The sprayer does work next to the vines so we have to either drive left or right from the vines.
+The sprayer does work next to the vines so it have to either drive left or right from the vines.
```
Comment (schwiti6190) [inline translations/translation_en.xml:500]: have -> has

File: translations/translation_fr.xml [hunk 2]
```diff
@@ -317,15 +317,112 @@
...
+		<text name="CP_help_page_courseGeneratorBaseFunctions_text"										text="
+Basics:
+The first few values you see here, are real basic stuff.
+- Work width: Most tools get the exact work width detected and you won't have to care about this setting at all. Adjusting it, will have an impact on your overall course.
+- Multiple Tools: This setting is used when you want more then just one vehicle to work on your course. As this gets a bit more complicated, there is a separate help topic for it.
+- Number of Headlands: The best way to keep your vehicles on the field while turning at the end of each row is to add headlands.
+The Number of the Headlands multiplied by the work width should be at least the total length of your vehicle plus the attached tool.
+Center options:
+- Field center: There are different modes on how your field center pattern should look like. The classic and most used one is up/down.
+Spiral, Racetrack and Lands have their own specific advantages over the others. Lands for example will have the combine's pipe point out of the fruit most of the time to easier unload them.
```
Comment (zallek) [inline translations/translation_fr.xml:366]: Could be nice to explain a bit more the purpose, pros and cons of each mode.

**Review by zallek — COMMENTED** [review]
Thanks a lot. It will greatly help players who are discovering CoursePlay like me 🎉

File: translations/translation_fr.xml (hunk 2 above)
Comment (Tensuko) [reply translations/translation_fr.xml:366]: Thanks for looking into it and give feedback.
I don't want to have it too long, but I take your feedback and try to figure something out.

**Pull request merged**